### ISSUE TITLE
Big form not working in recent versions

### ISSUE THREAD

**Issue by lognaturel** [getodk/collect#1001]
[giant-form.zip](https://github.com/opendatakit/collect/files/971376/giant-form.zip) used to work. It's really huge so depending on the device it can be really slow to load and use but I have just verified that I can get it going on my Nexus 5 running Android 6.0.1 and the 1.4.9 version downloaded from [here](https://opendatakit.org/downloads/download-info/odk-collect-apk/#).

In the current version (1.6.0), the form doesn't show up in the form list in Fill Blank Form. The top status bar shows that it's still scanning for forms but logcat doesn't show any activity. I have verified that this is also the case in 1.4.14 downloaded from [here](https://opendatakit.org/downloads/download-info/odk-collect-apk/#). 

One good first step for identifying the source of this issue would be to try to find the first version that has it.

**Comment by lognaturel** [comment]
Whaaaat? Now it opens just fine on 1.6.0. Would be good to get a repro attempt.

**Comment by mmarciniak90** [comment]
This form is really big - 7.4MB!
I tried to reproduce this problem on 3 devices. I used devices with Android v4.4, v5.1 and v6.0.
I verified 3 versions of Collect app : v1.4.9, v1.6.0 and v1.11.0.
All Collect versions were able to load this form and I was able to fill it. 
Loading this form takes some time but form was visible on Fill Blank Form view on all mentioned Collect versions and devices.
